### PR TITLE
Various updates to dashboard rendering.

### PR DESCRIPTION
**Pull request by yanniboi**

### CODE REVIEW

**Comment by yanniboi** [comment]
## Roles and Names update
 ### Before

selection_007


### After

selection_008

File: contacts.module [hunk 1]
```diff
@@ -253,7 +275,35 @@ function contacts_dashboard_block_edit_link_pre_render(array $element) {
...
+  $entity_types['profile']->setFormClass('contacts_dashboard', ContactsProfileForm::class);
+  $entity_types['profile']->setFormClass('user_dashboard', ContactsProfileForm::class);
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_insert() for profile_type.
+ *
+ * Set up entity displays for contacts and user dashboards.
+ */
+function contacts_profile_type_insert($entity) {
```
Comment (andrewbelcher) [inline contacts.module:288]: Do we want to assume custom displays for every profile type, or should we assume default and let site builders opt into custom?

**Review by andrewbelcher — COMMENTED** [review]
One question, other than that, looks great!

File: contacts.module (hunk 1 above)
Comment (yanniboi) [reply contacts.module:288]: Updated.

**Comment by andrewbelcher** [comment]
Latest change looks good!

**Pull request merged**